src/client/pages/OfferDebugger/components/QuoteData.tsx

```diff
@@ -132,28 +132,34 @@ const getCurrentAvailableQuoteData = (
   const currentQuoteTypeData = marketQuoteTypes.find(
     ({ value }) => value === currentQuoteType,
   )
-  return currentQuoteTypeData
-}
 
-const getDanishQuoteValues = (
-  values: any,
-  quoteType: 'danishTravelData' | 'danishAccidentData',
-) => {
-  const { danishHomeContentsData, ...filteredValues } = values
-  const { subType, livingSpace, ...quoteTypeValues } = danishHomeContentsData!
+  const randomId = Math.random()
+    .toString(36)
+    .substr(2, 5)
+  const email = `sven.svensson.${randomId}@hedvig.com`
+
   return {
-    ...filteredValues,
-    [quoteType]: quoteTypeValues,
+    ...currentQuoteTypeData,
+    initialFormValues: {
+      ...currentQuoteTypeData?.initialFormValues,
+      email,
+    },
   }
 }
 
+const getDanishQuoteDetailValues = (values: any) => {
+  const { subType, livingSpace, ...quoteTypeValues } = values.data
+  return quoteTypeValues
+}
+
 const getSwedishAccidentQuoteValues = (values: any) => {
-  const { swedishApartmentData, ...filteredValues } = values
-  const { subType, ...quoteTypeValues } = swedishApartmentData!
+  const { data: swedishApartmentData, ...filteredValues } = values
+  const { subType, ...quoteTypeValues } = swedishApartmentData
   return {
     ...filteredValues,
-    swedishAccidentData: {
+    data: {
       ...quoteTypeValues,
+      type: QuoteType.SwedishAccident,
       student:
```

Comment:
Do you know if it's in scope to change the student types for SE as well? Would be so nice to not care about StudentBrf and StudentRent :)

Reply:
aha, nope I haven't heard anyone talking about that - maybe ask in #memberless-onboarding-api